plone/recipe/codeanalysis/__init__.py

```diff
@@ -77,6 +79,9 @@ def __init__(self, buildout, name, options):
         # CSS Lint
         self.options.setdefault('csslint', 'False')
         self.options.setdefault('csslint-bin', 'csslint')
+        # check-manifest
+        self.options.setdefault('check-manifest', 'True')
```

Comment:
Although I agree that it should be always checked, I'm not sure if we want to enable by default...